python/src/aac/plugins/check/check_aac.aac

```diff
@@ -16,4 +16,4 @@ plugin:
           description: The check command will fail if encountering fail or warn constraint violations.
         - name: --verbose
```

Comment:
Of course it won't let me highlight all of it -.-

I don't believe the unit tests exercised the flags available with this command, did they? --fail-on-warn and --verbose.

Would like to see those included for the unit test and see if it is possible to incorporate to the behave tests as well.